master/buildbot/master.py

```diff
@@ -93,6 +93,8 @@ def __init__(self, basedir, configFileName="master.cfg", umask=None):
         self.reconfig_active = False
         self.reconfig_requested = False
         self.reconfig_notifier = None
+        self.change_services_state = False
```

Comment:
@campos-ddc this is not just for reconfiguring but also when the state of the services changes I'm ok with is_changing_service to be more general